### PR TITLE
Better illustration

### PR DESCRIPTION
**Pull request by prashanthmadi**
many of them i know have multiple subscriptions with same name. It would great if we make a note of ability to set default account using subscriptionId

### CODE REVIEW

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md](https://github.com/prashanthmadi/azure-docs-cli-python/blob/patch-1/docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/manage-azure-subscriptions-azure-cli?branch=pr-en-us-302) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C11%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201704110326577215-302%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md](https://github.com/prashanthmadi/azure-docs-cli-python/blob/patch-1/docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/manage-azure-subscriptions-azure-cli?branch=pr-en-us-302) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C28%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201706282118486219-302%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md [hunk 1]
```diff
@@ -41,5 +41,6 @@ If so, you can configure Azure CLI 2.0 to execute commands against a particular
    ```
 
 You can verify the change by running the `az account list --output table` command again.
+Note : You can also use SubscriptionId instead of Subscription Name to set default account.
```
Comment (sptramer) [inline docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md]: The `NOTE:` designation should be removed, and the wording should be changed to `You can use either the subscription ID or name to set the default account."

I would recommend that this line be moved to before the "You can verify...".

**Comment by sptramer** [comment]
@prashanthmadi Please make the requested changes and rebase the current `master` into your branch if you would still like this PR to be evaluated and merged.

**Comment by sptramer** [comment]
Closing due to lack of activity and conflicts. @prashanthmadi if you would like the changes to be merged, please update accordingly.

**Pull request closed without merging**